pkgs/servers/search/elasticsearch/plugins.nix

```diff
@@ -36,6 +36,21 @@ in {
     };
   };
 
+  elasticsearch_cloud_aws = esPlugin rec {
+    name = "elasticsearch-cloud-aws-${version}";
+    pluginName = "elasticsearch-cloud-aws";
+    version = "2.4.2";
+    src = fetchurl {
+      url = "https://github.com/elastic/elasticsearch-cloud-aws/archive/v2.4.2.zip";
```

Comment:
It's not about the ref vs. the hash. It's that the archive it constructs literally changes, even for the exact same hash. I get the impression they cache a generated one for a while, then it expires and you get a new file.